src/nbl/builtin/utils.cmake

```diff
@@ -0,0 +1,91 @@
+macro(LIST_BUILTIN_RESOURCE _BUNDLE_NAME_ _LBR_PATH_)
+	set(_OPTIONAL_ALIAS_ "${ARGV2}")
+	
+	list(APPEND ${_BUNDLE_NAME_} "${_LBR_PATH_}")
+	set(${_BUNDLE_NAME_} ${${_BUNDLE_NAME_}} PARENT_SCOPE) # override
+	
+	list(APPEND _LBR_${_BUNDLE_NAME_}_ "${_LBR_PATH_},${_OPTIONAL_ALIAS_}")
+	set(_LBR_${_BUNDLE_NAME_}_ ${_LBR_${_BUNDLE_NAME_}_} PARENT_SCOPE) # override
```

Comment:
?

Reply:
we need to have updated list for each bundle with parent scope after each `LIST_BUILTIN_RESOURCE` invocation